plugins/connections/ui/src/main/java/org/pentaho/di/vfs/connections/ui/dialog/ConnectionDialog.java

```diff
@@ -410,7 +426,8 @@ private void test() {
       boolean result = false;
       MessageBox mb;
       try {
-        result = connectionManager.test( connectionDetails );
+        boolean ignoreRootPath = StringUtils.isEmpty( connectionDetails.getRootPath() );
```

Comment:
Imo, this parameter should always be `false`.
PDI shows all connection information, including the root path, on a single page, and so does not need to ignore the root path.

Reply:
When no root path is provided by the user, and on sending the above flag 'true', when empty root path is provided, thought of returning from the code without further validation of root path. Hence added the above flag.
![image](https://github.com/pentaho/pentaho-kettle/assets/133358829/ae843761-9689-4f18-ae76-e64d017544b1)

Reply:
If the root path is required and the user does not fill it in, the UI must get a not valid result and warn the user the connection is not valid.

Reply:
Addressed